platiagro/datasets.py

```diff
@@ -133,7 +133,12 @@ def save_dataset(name: str,
 
     try:
         # gets metadata (if dataset exists)
-        metadata = stat_dataset(name, run_id)
+        stored_metadata = stat_dataset(name, run_id)
+        # update stored metadata values
+        if metadata:
+            for key, value in metadata.items():
+                stored_metadata[key] = value
```

Comment:
@shiguemi32 existe uma função do dict que faz isso sem precisar do for:
https://docs.python.org/3/library/stdtypes.html#dict.update

Reply:
Feito